src/frontend/src/views/LandingPageV2/components/Footer/index.tsx

```diff
@@ -1,6 +1,6 @@
 import Image from "next/image";
 import { useEffect } from "react";
-import BiohubLogoImg from "src/common/images/cz-biohub-logo.png";
+import BiohubLogoImg from "src/common/images/biohub-sf-logo.svg";
```

Comment:
It seems like this is the only use of the old logo in the repo. What do you think about removing it?